monitoring/mock_uss/interuss_logging/routes_interuss_log.py

```diff
@@ -0,0 +1,83 @@
+from typing import Tuple
+import yaml
+import os
+from implicitdict import ImplicitDict, StringBasedDateTime
+from flask import request, abort, jsonify
+from yaml.constructor import ConstructorError
+from loguru import logger
+
+from monitoring.mock_uss import webapp
+from monitoring.mock_uss.auth import requires_scope
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import (
+    SCOPE_SCD_QUALIFIER_INJECT,
+)
+from monitoring.monitorlib.mock_uss_interface.interuss_interaction import (
+    Interaction,
+    Issue,
+)
+
+from monitoring.mock_uss.interuss_logging.config import KEY_LOG_DIR
+
+
+@webapp.route("/mock_uss/interuss/log", methods=["GET"])
```

Comment:
The base URL should be named for the functionality.  So, currently:

```suggestion
@webapp.route("/mock_uss/interuss_logging/log", methods=["GET"])
```

Reply:
Resolved.